data-prepper-plugins/http-common/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'java'
+}
+
+repositories {
```

Comment:
You don't need this block. And we are moving away from declaring repositories here. See #4152 and #4161. Please remove this block.